telegram/_inline/inlinequeryresultaudio.py

```diff
@@ -67,15 +63,11 @@ class InlineQueryResultAudio(InlineQueryResult):
         caption (:obj:`str`): Optional. Caption,
             0-:tg-const:`telegram.constants.MessageLimit.CAPTION_LENGTH` characters after entities
             parsing.
-        parse_mode (:obj:`str`): Optional. Send Markdown or HTML, if you want Telegram apps to show
-            bold, italic, fixed-width text or inline URLs in the media caption. See the constants
-            in :class:`telegram.constants.ParseMode` for the available modes.
-        caption_entities (List[:class:`telegram.MessageEntity`]): Optional. List of special
-            entities that appear in the caption, which can be specified instead of
-            :paramref:`parse_mode`.
-        reply_markup (:class:`telegram.InlineKeyboardMarkup`): Optional. Inline keyboard attached
+        parse_mode (:obj:`str`, optional): |parse_mode|
+        caption_entities (List[:class:`telegram.MessageEntity`], optional): |caption_entities|
+        reply_markup (:class:`telegram.InlineKeyboardMarkup`, optional): Inline keyboard attached
```

Comment:
```suggestion
        parse_mode (:obj:`str`): Optional. |parse_mode|
        caption_entities (List[:class:`telegram.MessageEntity`]): Optional. |caption_entities|
        reply_markup (:class:`telegram.InlineKeyboardMarkup`): Optional. Inline keyboard attached
```
the Attributes section should have Optional after the brackets. Can you apply these changes in all the files?

Reply:
Ah should have asked before, I've followed the example from [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) (see `ExampleClass`), as mentioned in the contribution guidelines :)

Reply:
Actually there's a bunch of places where `, optional):` is used (e.g. [here](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/telegram/_inline/inlinequery.py#L75)). Do you want me to revert those ones as well?

Reply:
Yep. Those are wrong, so you can change them too :)